cartoframes/data/services/isolines.py

```diff
@@ -146,6 +146,7 @@ def _iso_areas(self,
         else:  # source.is_local()
             # upload to temporary table
             temporary_table_name = self._new_temporary_table_name()
+            compute_geodataframe(source)
```

Comment:
Couldn't we rely on `upload` converting lng/lat to geometry on the fly? I recall there's a `with_lnglat` option in the method that generates CSV data to be copied but I don't remember if that works automatically.

Reply:
There is no automatic detection for the upload, because you need to pass explicitly the parameters.